### PR TITLE
fix vocabulary permissions for non-registered vocabs by providing a default

### PR DESCRIPTION
**Pull request by jensens**
should fix current broken coredev tests

### CODE REVIEW

**Comment by thet** [comment]
Yes! Jenkins going green again!

File: plone/app/content/browser/vocabulary.py [hunk 1]
```diff
@@ -1,41 +1,48 @@
...
-_permissions = {
+DEFAULT_PERMISSION = 'View'
+PERMISSIONS = {
     'plone.app.vocabularies.Users': 'Modify portal content',
     'plone.app.vocabularies.Catalog': 'View',
     'plone.app.vocabularies.Keywords': 'Modify portal content',
     'plone.app.vocabularies.SyndicatableFeedItems': 'Modify portal content'
 }
 
+_permissions = deprecated(PERMISSIONS, 'Use PERMISSION variable instead')
```
Comment (mauritsvanrees) [inline plone/app/content/browser/vocabulary.py]: The `deprecated` call cannot handle a dictionary and returns `None`. So `_permissions` is `None` after this call. I noticed this in ploneintranet, which imported `_permissions` and got into problems because it was `None`.

`zope.deprecated` handle strings, tuples, lists, methods, and functions. Not dictionaries. I tried adding that, but you run into an error `cannot hash dictionary`. The list which is does support, also cannot be just any list, but is meant to be a list of strings which are variable names that have been imported or defined in the current module.
In other words, this is not the way. :-)

Fixed in pull request #95.
